src/Faker/Provider/Company.php

```diff
@@ -27,4 +28,12 @@ public static function companySuffix()
     {
         return static::randomElement(static::$companySuffix);
     }
+
+    /*
+     * @example 'Cashier'
+     */
+    public static function jobTitle()
```

Comment:
Can't be static, because of `$this`. A unit tests would have showed it to you. This shows me that you haven't tested you own code, it's bad. Please add a unit test.